docs/NEO4J_QUERY_PATTERNS.md

```
@@ -0,0 +1,443 @@
+
+```cypher
+MATCH (d:Developer)<-[:PERFORMED_BY]-(i:Investigation)
+OPTIONAL MATCH (i)-[:CONTAINS|DISCOVERED_FINDING]->(f)
+WITH d, i, count(DISTINCT f) as findings
+RETURN d.name as developer,
+       count(DISTINCT i) as investigations,
+       sum(findings) as total_findings,
+       collect(DISTINCT labels(f)[0]) as finding_types
+ORDER BY investigations DESC
+```
+
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)